doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc

```diff
@@ -0,0 +1,55 @@
+[id='bus-app-async-execution-proc_{context}']
+= Configuring your Sping Boot application for asynchronous execution
+
+After you create your Spring Boot project, you can use the `jbpm.executor` parameters to enable asynchronous execution.
+
+.Prerequisites
+* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+
+.Procedure
+. Extract the ZIP file and navigate to the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` folder, where `<BUSINESS-APPLICATION>` is the name of your Spring Boot project.
+. Open the `application.properties` file in a text editor.
+
+. To enable asynchronous execution, set the value of the `jbpm.executor.enabled` parameter to `true`, uncomment the other `jbpm.executor` parameters, and change the values as required, for example:
```

Comment:
Probably my bad, but I would have written this as "and change the values as required. For example:"

Reply:
Yeah, both are wrong, actually. Should be:
as required, as shown in the following example:  (IBM SG)

For what it's worth, I prefer your version.